packages/terra-clinical-label-value-view/src/LabelValueView.scss

```diff
@@ -7,7 +7,7 @@
   }
 
   .label {
-    color: #75767d;
+    color: var(--terra-label-value-label-color, #75767d);
```

Comment:
Update the variable to --terra-label-value-view-label-color;
@neilpfeiffer Can you verify?